lib/engines/hcktest/playlist.rb

```diff
@@ -52,36 +57,61 @@ def ms_playlist(log)
       workspace_file
     end
 
+    def custom_playlist
+      playlist = @project.options.test.playlist
+
+      workspace_file = "#{@project.workspace_path}/playlist_#{@kit[3..]}.xml"
+
+      FileUtils.cp(playlist, workspace_file)
+
+      @logger.info("Applying custom kit playlist from #{playlist}")
+      workspace_file
+    end
+
     def sort_by_duration
       @tests.each_with_index do |x, i|
         @tests[i]['duration'] = time_to_seconds(x['estimatedruntime'])
       end
       @tests.sort_by! { |test| test['duration'] }
     end
 
-    def custom_playlist(log)
-      playlist = @project.engine.target['playlist']
-      return unless playlist
+    def custom_select_test_names(log)
+      user_test_names_file = @project.options.test.select_test_names
```

Comment:
Include the word `select` for consistency.

```suggestion
      user_select_test_names_file = @project.options.test.select_test_names
```